src/firmware/main/app/world/firmware_robot.h

```diff
@@ -59,19 +103,55 @@ Chicker_t* app_firmware_robot_getChicker(FirmwareRobot_t* robot);
 Dribbler_t* app_firmware_robot_getDribbler(FirmwareRobot_t* robot);
 
 /**
- * Get the x-position of the given ball
- * @param ball The ball to get the y-position for
- * @return The x-position of the given ball, in meters, in global coordinates
+ * Get the x-position of the given robot
+ * @param robot The robot to get the y-position for
+ * @return The x-position of the given robot, in meters, in global coordinates
  */
 float app_firmware_robot_getPositionX(FirmwareRobot_t* robot);
 
 /**
- * Get the y-position of the given ball
- * @param ball The ball to get the y-position for
- * @return The y-position of the given ball, in meters, in global coordinates
+ * Get the y-position of the given robot
+ * @param robot The robot to get the y-position for
+ * @return The y-position of the given robot, in meters, in global coordinates
  */
 float app_firmware_robot_getPositionY(FirmwareRobot_t* robot);
 
+/**
+ * Get the current orientation of the given robot
```

Comment:
We should specify what coordinate frame all of this is in.

Reply:
see above comment